torch/testing/_internal/common_methods_invocations.py

```diff
@@ -4177,6 +4189,47 @@ def gradcheck_wrapper_triangular_input(op, input, *args, upper=False, **kwargs):
                        SkipInfo('TestUnaryUfuncs', 'test_reference_numerics_hard',
                                 device_type='cpu', dtypes=[torch.complex64])
                    )),
+    OpInfo('__radd__',
+           op=torch.Tensor.__radd__,
+           dtypes=all_types_and_complex_and(torch.bfloat16, torch.half, torch.bool),
+           sample_inputs_func=sample_inputs_rbinops,
+           supports_out=False,
+           skips=(SkipInfo('TestCommon', 'test_variant_consistency_jit',),),
```

Comment:
Skipping `test_variant_consistency_jit` here because if I don't, I get the following error when running `test_ops.py` tests:

```
object has no attribute __radd__:
  File "<string>", line 3

def the_method(i0):
    return torch.__radd__(i0, 3.14)
           ~~~~~~~~~~~~~~ <--- HERE
```

Changing the name to `OpInfo('Tensor.__radd__'` changes the error to (even though calling `torch.Tensor.__radd__(torch.tensor(0), 1)` works):

```
attribute lookup is not defined on builtin:
  File "<string>", line 3

def the_method(i0):
    return torch.Tensor.__radd__(i0, 3.14)
           ~~~~~~~~~~~~~~~~~~~~~ <--- HERE
```

Not sure why that happens...

Reply:
Thanks for noticing this, @ysiraichi; skipping this test for now seems like the right idea